Detectors/TPC/base/include/TPCBase/CalArray.h

```diff
@@ -247,7 +247,9 @@ inline const CalArray<T>& CalArray<T>::operator/=(const CalArray<T>& other)
     return *this;
   }
   for (size_t i = 0; i < mData.size(); ++i) {
-    mData[i] /= other.getValue(i);
+    if (other.getValue(i) != 0) {
+      mData[i] /= other.getValue(i);
+    }
```

Comment:
Perhaps you should print an error stating that division by 0 was detected and set the value to 0